File: token-validation-spring-demo/src/test/java/no/nav/security/token/support/demo/spring/LocalSecurityConfiguration.java
package no.nav.security.token.support.demo.spring;

import no.nav.security.token.support.spring.test.EnableMockOAuth2Server;
import org.springframework.context.annotation.Configuration;
import org.springframework.context.annotation.Profile;

@Configuration
@Profile("local")
@EnableMockOAuth2Server
public class LocalSecurityConfiguration {

}

File: token-validation-spring-demo/src/test/kotlin/no/nav/security/token/support/demo/spring/LocalSecurityConfiguration.kt
package no.nav.security.token.support.demo.spring

import org.springframework.context.annotation.Configuration
import org.springframework.context.annotation.Profile
import no.nav.security.token.support.spring.test.EnableMockOAuth2Server

@Configuration
@Profile("local")
@EnableMockOAuth2Server
class LocalSecurityConfiguration